storage/src/journal/fixed.rs

```diff
@@ -1041,4 +1139,68 @@ mod tests {
             assert_eq!(journal.oldest_retained_pos().await.unwrap(), None);
         });
     }
+
+    #[test_traced]
+    fn test_journal_conformance() {
```

Comment:
add a function doc?  unclear (to me) what this is doing from the name alone.

Reply:
(This is to protect against silent changes to the storage format)